### PR TITLE
[TG-54] Added pay now button

### PR DESCRIPTION
**Pull request by lychanl**

### CODE REVIEW

**Comment by Delebrith** [comment]
If any of you @Grushenko @Swianka @gala377 accept those changes, please merge the PR.

File: apsi-enrollment-fe/src/app/modules/main/pages/event-detail/event-detail.component.ts [hunk 1]
```diff
@@ -39,22 +40,28 @@ export class EventDetailComponent implements OnInit {
...
-        }
-      );
+    this.createPayment(this.eventService.signUp(this.event.id));
+  }
+
+  onPayNow() {
+    this.createPayment(this.eventService.getEnrollment(this.event.id));
+  }
+
+  createPayment(enrollment: Observable<Enrollment>) {
```
Comment (Grushenko) [inline apsi-enrollment-fe/src/app/modules/main/pages/event-detail/event-detail.component.ts:50]: In most cases observable should not be argument of a function. You should use `switchMap`.

Comment (lychanl) [reply apsi-enrollment-fe/src/app/modules/main/pages/event-detail/event-detail.component.ts]: changed

**Comment by lychanl** [comment]
@Grushenko unfortunately that did exactly what I have tried to avoid by using this function i.e. duplicated code

**Pull request merged**